distribution/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -138,3 +138,892 @@ For example, if a Spatial search and Contextual search are the components of a C
+(The amount of time that a "long pause" is defined as is configurable, with the default value being five seconds.)
+The Catalog Framework will attempt to retrieve the product up to a configurable number of times (default = three), waiting for a configurable amount of time (default = 10 seconds) between each attempt, trying to successfully retrieve the product.
+If the Catalog Framework is unable to retrieve the product, an error message is returned to the client.
+
+If the admin has enabled the *Always Cache When Canceled* option, caching of the product will occur even if the client cancels the product retrieval so that future requests will be serviced quickly.
+Otherwise, caching is canceled if the user cancels the product download.
+
+===== Product Download Status
+
+As part of the caching of products, the Catalog Framework also posts events to the OSGi notification framework.
+Information includes when the product download started, whether the download is retrying or failed (after the number of retrieval attempts configured for product caching has been exhausted), and when the download completes.
+These events are retrieved by the Search UI and presented to the user who initiated the download.
+
+===== Notifications and Activities
+
+====== Notifications
+
+Currently, the notifications provide information about product retrieval only.
+For example, in the ${branding} Search UI, after a user initiates a resource download, they receive notifications when the download completed, failed, canceled, or is being retried.
+
+====== Activities
+
+Activities can be enabled by selecting "Show tasks" in the Standard Search UI configuration.
+Activity events include the status and progress of actions that are being performed by the user, such as searches and downloads.
+A list of all activities opens in a drop-down menu, from which activities can be read and deleted.
+If a download activity is being performed, the Activity drop-down menu provides the link to retrieve the product.
+If caching is enabled, a progress bar is displayed in the Activity (Product Retrieval) drop-down menu until the action being performed is complete.
+
+===== Managing Asynchronous Capabilities
+
+====== Configuring the Product Cache
+
+All caching properties are part of the Catalog Framework Configuration
+
+[cols="1m,1,1,1,1" options="header"]
+|===
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|productCacheDirectory
+|String
+|Directory where retrieved products will be cached for faster, future retrieval. If a directory path is specified with directories that do not exist, Catalog Framework will attempt to create those directories. Without configuration, the product cache directory is `<INSTALL_DIR>/data/productcache`. If a relative
+path is provided it will be relative to the `<INSTALL_DIR>`.
+
+It is recommended to enter an absolute directory path such as `/opt/productcache` in Linux or `C:/product-cache` in Windows.
```

Comment:
Should it be `C:\product-cache`? Not sure if DDF can switch between slashes in windows...